### PR TITLE
Add a binder link and a notebook example to compute glider metrics

### PR DESCRIPTION
**Pull request by ocefpaf**
@kbailey-noaa this is a draft PR with a binder link and example for your use case.

- [x] binder link
- [x] create example with the example script

### CODE REVIEW

**Comment by kerfoot** [comment]
@ocefpaf pushed commits. Example script is: https://github.com/kerfoot/gdutils/blob/master/scripts/examples/search_datasets.py

**Comment by ocefpaf** [comment]
Folks you can try this note book now with this link: [Binder](https://mybinder.org/v2/gh/ocefpaf/gdutils/binder?filepath=notebooks)

I'll finish up some details, it should be good to go by Monday.

**Comment by kerfoot** [comment]
Nevermind, seems to work now.

@ocefpaf Got this error when I clicked on the binder link:

Could not resolve ref for gh:kerfoot/gdutils/binder. Double check your URL. GitHub recently changed default branches from "master" to "main".

File: .binder/environment.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: gdutils
```
Comment (ocefpaf) [inline .binder/environment.yml:1]: @kerfoot I reduced your env file to a bare minimum and created a lock file so binder starts faster. My next PR will be to package gdutils but you may want to save your current environment file until we do so.

**Comment by ocefpaf** [comment]
@kerfoot this is ready for a review. Please let me know if you need anything.

I'll be working on making gdutils a package today.

**Comment by ocefpaf** [comment]
Closing this. If this is something you still want to do we should start from a fresh PR.

**Pull request closed without merging**